packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -54,18 +56,21 @@ export default function ProfileMenu({
   const { openModal } = useLazyModal();
   const { user, logout } = useAuthContext();
   const { isActive: isDndActive, setShowDnd } = useDndContext();
+  const showPlusSubscription = useFeature(feature.plusSubscription);
```

Comment:
Think you need to useConditionalFeature to only enrol when menu is active.

Reply:
Don't think it's needed on this one. `ProfileMenu` is for desktop, and is only rendered when `ProfileButton` is clicked?

Reply:
If thats the case then its fine